modules/041-linstor/templates/sds-drbd-operator/secret.yaml

```diff
@@ -0,0 +1,20 @@
+---
+{{ define "sds_drbd_data_nodes_node_selector" }}
+nodeSelector:
+  {{- . | toYaml | nindent 2 }}
+{{- end }}
+  ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: d8-sds-drbd-operator-config
+  namespace: d8-{{ .Chart.Name }}
+type: Opaque
+stringData:
+  config: |
+{{- if dig "dataNodes" "nodeSelector" false .Values.linstor }}
```

Comment:
There should be default value for `linstor.dataNodes.nodeSelector` = `kubernetes.io/os: linux` in openapi.